style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  width: fit-content;
+  height: fit-content;
+  background: #e5e5e5;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.part1 {
+  border: 1px solid rgb(202, 199, 199);
+  background: url(background.jpeg);
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
```

Comment:
- [ ] Nice work implementing the design :clap: However it would be nice to use the same color as the figma design (white), please follow the Figma design as requested
### Your design :point_down: 
![Screenshot from 2023-05-29 09-19-41](https://github.com/mumbereh/my-portfolio/assets/91142432/4aae9bc5-0c52-4cdd-81db-cd026faf6a47)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-04-08 19-44-56](https://github.com/mumbereh/my-portfolio/assets/91142432/9f1a6ab3-25c2-4b3b-85aa-50dda319eb54)